packages/gasket-plugin-command/README.md

```diff
@@ -24,212 +25,220 @@ export default makeGasket({
 });
 ```
 
+---
+
 ## Lifecycles
 
-### Commands
+### `commands`
+
+The `commands` lifecycle is executed during the `configure` hook if the `gasket`
+CLI command is present in the `argv`. You can define commands that include
+arguments, options, and custom parsing logic.
+
+#### Examples
 
-Executed in the `configure` hook of `@gasket/plugin-command` if the `gasket` command is present in the `argv`. The `gasket` command is the CLI execution of the `gasket.js` file.
+##### **Basic Command**
```

Comment:
To avoid the headings getting so deep, and the extra `**` here, just include "Example" in the heading description

```diff
- #### Examples
- ##### **Basic Command**
+ #### Example Basic Command
```